### PR TITLE
Docs: Document migration of dashboard alert permissions

### PR DESCRIPTION
**Pull request by papagian**
**What this PR does / why we need it**:
It updates Grafana 8 Alerts [opt-in](https://grafana.com/docs/grafana/latest/alerting/unified-alerting/opt-in/) page by providing additional information about permissions.
Also it removed warning for beta.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/alerting/unified-alerting/opt-in.md [hunk 1]
```diff
@@ -10,7 +10,13 @@ Setting the `ngalert` feature toggle enables the new Grafana 8 alerting system.
 
 >**Note:** It is recommended to backup Grafana's database before enabling this feature. If you are using PostgreSQL as the backend data source, then the minimum required version is 9.5.
 
-At startup, when the feature toggle is enabled, Grafana dashboard alerting is disabled and existing dashboard alerts are migrated into a format that is compatible with the Grafana 8 alerting system. You can view these migrated rules, alongside any new alerts you create after the migration, from the Alerting page of your Grafana installation. 
+At startup, when the feature toggle is enabled, Grafana dashboard alerting is disabled and existing dashboard alerts are migrated into a format that is compatible with the Grafana 8 alerting system. You can view these migrated rules, alongside any new alerts you create after the migration, from the Alerting page of your Grafana installation.
+
+Read and write access to dashboard alerts used to be governed by the dashboard and folder permissions under they lived. Grafana 8 Alerts also live under folders and inherite their permissions. During the migration dashboard alert permissions are matched to the new rules permissions as follows:
```
Comment (pkolyvas) [inline docs/sources/alerting/unified-alerting/opt-in.md:15]: Small suggestion & correction:

`Read and write access to dashboard alerts in Grafana versions 7 and earlier were governed by the dashboard and folder permissions under which the the alerts were stored. In Grafana 8, alerts are also stored in folders and inherit the permissions of those folders. During the migration, dashboard alert permissions are matched to the new rules permissions as follows:`

**Review by pkolyvas — CHANGES_REQUESTED** [review]
Made a couple of suggestions for clarity.

File: docs/sources/alerting/unified-alerting/opt-in.md [hunk 2]
```diff
@@ -10,7 +10,13 @@ Setting the `ngalert` feature toggle enables the new Grafana 8 alerting system.
 
 >**Note:** It is recommended to backup Grafana's database before enabling this feature. If you are using PostgreSQL as the backend data source, then the minimum required version is 9.5.
 
-At startup, when the feature toggle is enabled, Grafana dashboard alerting is disabled and existing dashboard alerts are migrated into a format that is compatible with the Grafana 8 alerting system. You can view these migrated rules, alongside any new alerts you create after the migration, from the Alerting page of your Grafana installation. 
+At startup, when [the feature toggle is enabled]({{< relref "../../administration/configuration.md">}}#feature_toggles), Grafana dashboard alerting is disabled and existing dashboard alerts are migrated into a format that is compatible with the Grafana 8 alerting system. You can view these migrated rules, alongside any new alerts you create after the migration, from the Alerting page of your Grafana installation.
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/opt-in.md:13]:
```suggestion
At startup, when [the feature toggle is enabled]({{< relref "../../administration/configuration.md">}}#feature_toggles), the legacy Grafana dashboard alerting is disabled and existing dashboard alerts are migrated into a format that is compatible with the Grafana 8 alerting system. You can view these migrated rules, alongside any new alerts you create after the migration, from the Alerting page of your Grafana instance.
```

File: docs/sources/alerting/unified-alerting/opt-in.md [hunk 3]
```diff
@@ -10,7 +10,13 @@ Setting the `ngalert` feature toggle enables the new Grafana 8 alerting system.
...
+
+Read and write access to dashboard alerts in Grafana versions 7 and earlier were governed by the dashboard and folder permissions under which the the alerts were stored. In Grafana 8, alerts are also stored in folders and inherit the permissions of those folders. During the migration, dashboard alert permissions are matched to the new rules permissions as follows:
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/opt-in.md:15]:
```suggestion
Read and write access to dashboard alerts in Grafana versions 7 and earlier were governed by the dashboard and folder permissions under which the alerts were stored. In Grafana 8, alerts are stored in folders and inherit the permissions of those folders. During the migration, dashboard alert permissions are matched to the new rules permissions as follows:
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added some minor change requests.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Review by pkolyvas — APPROVED** [review]
Thank you!

**Pull request merged**